guacamole/src/main/webapp/app/index/styles/ui.css

```diff
@@ -150,6 +150,12 @@ div.section {
     padding: 0.1em;
 }
 
+.list-item .caption:after {
```

Comment:
There is nothing wrong with these CSS changes themselves, but I have to say, the word "clearfix" makes me squirm. It harks back to an ancient hack and code riddled with `<div class="thingy clearfix"> ... </div>`.

You're clearing the float with a pseudo element. Sounds reasonable.